amida-tech/indaba-client
# Bug/inba 977 jf select fixes

#### What does this PR do?
Fixes the react-select components that were broken as of INBA-953.

#### Related JIRA tickets:
INBA-953.

#### How should this be manually tested?
There are four areas to check:

In Project Management, play with the "User Sidebar" on the right, choosing different groups in a project with multiple groups. Don't forget to try it in tandem with the user filter just above it to ensure it's working correctly. 

In Profile, ensure that choosing a different notification type sticks when you reload the page.

Click on the ellipsis (...) for a Task. Ensure that listing of users you can reassign it to loads and works. There are other tickets/future tickets associated with better UI controls for this so please don't report on that. We already know. Just focus on the select.

Finally, assign a few "Dropdown" type questions to a survey (I recommend at least two). As the admin, go into the Task Review to ensure that the disabled functionality for those dropdowns work. As an assigned user, click and test that those dropdowns work correctly. You want two because you also want to see one _without_ any chosen answers.

#### Background/Context

#### Screenshots (if appropriate):
